distribution/ddf-common/src/main/resources/etc/transformers/README.md

```diff
@@ -0,0 +1,23 @@
+# Summary
```

Comment:
Do we need some ask docs on this as well?

Reply:
Yes. We are going to want docs. 
FYI @ricklarsen